src/components/features/cart/components/CartModal.tsx

```
@@ -0,0 +1,71 @@
+  );
+
+  return (
+    <Modal.Container
+      title="장바구니"
+      showCloseButton={false}
+      position="bottom"
+      style={{ maxHeight: 'calc(100% - 120px)', overflow: 'auto' }}
+    >
+      {cartList.map(({ id, quantity, product }) => (
+        <Flex>
+          <Separator />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)